components/compliance-service/api/tests/64_A2V4_migration_spec.rb

```diff
@@ -0,0 +1,273 @@
+##### GRPC SETUP #####
+require 'api/reporting/reporting_pb'
+require 'api/reporting/reporting_services_pb'
+
+describe File.basename(__FILE__) do
+  Reporting = Chef::Automate::Domain::Compliance::Api::Reporting unless defined?(Reporting)
+
+  it "works" do
```

Comment:
I used the `it "works"` pattern because a lot of existing tests couldn't be broken up easily as later tests depended on data created in earlier tests. For a brand new test file it'd be better for readability and debugging to break these into individual tests that could fail independently.

Reply:
Thank you for pointing that out. I broke it up into many tests.